R/create_monthly_beddays.R

```diff
@@ -59,5 +60,14 @@ create_monthly_beddays <- function(data, year, admission_date, discharge_date, c
   data <- dplyr::bind_cols(data, beddays) %>%
     dplyr::select(-.data$stay_interval)
 
+  # Work out costs for each month if applicable
+  if (include_costs) {
+    costs <- beddays %>%
+      dplyr::rename_with(~ str_replace(., "_beddays", "_costs"))
+
+    data <- dplyr::bind_cols(data, costs) %>%
+      dplyr::mutate(dplyr::across(dplyr::ends_with("_costs"), ~ dplyr::if_else(yearstay != 0, .x / yearstay * cost_total_net, 0)))
```

Comment:
Sorry, realised that I suggested this, but should this not be:
```suggestion
    data <- dplyr::bind_cols(data, costs) %>%
      dplyr::mutate(dplyr::across(dplyr::ends_with("_costs"), ~ dplyr::if_else(.x!= 0, .x / yearstay * cost_total_net, 0)))
```
i.e. if beddays are zero then set cost to be zero, otherwise `beddays / yearstay * cost`